src/core/default/commands/relationaldb/shell.py

```diff
@@ -143,12 +145,20 @@ def __init__(
         self, fmt: fmt, cluster_arn: str, secret_arn: str, database_name: str
```

Comment:
The location of the history can use this 
```
history_location = os.path.join(Workspace.instance().settings.INTERMEDIATE_FOLDER_LOCATION, 'dbshell')

if not os.path.isfile(history_location):
    # touch the file
    with open(history_location, 'a'):
           pass
```

This will create the file in the .cdev/intermediate folder which is where files that can be delete go.